authenticate

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-ubuntu-personal-store.generate-token
+$SNAP/bin/generate-token
```

Comment:
This seems to fix the issue generating the token in Xenial. 
It looks like it's not happy with calling the command as it is outside the snap package.